content/en/apps/guides/hosting/4.x/app-developer.md

```diff
@@ -0,0 +1,144 @@
+* port for `HTTPS` traffic (`NGINX_HTTP_PORT`)
+* directory for storing the compose files and CouchDB files
+
+Assuming you want to start a new project called `the_second` and  start the instance on `HTTP` port `8081` and `HTTPS` port `8443`, we would first create a new directory and download the same files:
+
+```shell 
+mkdir  ~/the_second && cd ~/the_second
+curl -s -o docker-compose.yml https://raw.githubusercontent.com/medic/cht-upgrade-service/main/docker-compose.yml
+curl -s -o cht-core.yml https://staging.dev.medicmobile.org/_couch/builds_4/medic%3Amedic%3Amaster/docker-compose/cht-core.yml
+curl -s -o cht-couchdb.yml https://staging.dev.medicmobile.org/_couch/builds_4/medic%3Amedic%3Amaster/docker-compose/cht-couchdb.yml
+```
+
+
+Then, we would use the same `docker-compose` command as above, but this time specify the ports:
+
+```shell script
+NGINX_HTTP_PORT=8081 NGINX_HTTPS_PORT=8444 CHT_COMPOSE_PROJECT_NAME=app-devl COUCHDB_SECRET=foo DOCKER_CONFIG_PATH=${PWD} COUCHDB_DATA=${PWD}/couchd CHT_COMPOSE_PATH=${PWD} COUCHDB_USER=medic COUCHDB_PASSWORD=password docker-compose up
+```
+
+The second instance is now accessible at  [https://localhost:8444](https://localhost:8444) and again using username `medic` and password `password` to login.
+
+## The `.env` file
+
+Often times it's convenient to use revision control, like GitHub, to store and publish changes in a CHT app.  A nice compliment to this is to store the specifics on how to run the `docker-compose` command for each app. By using a shared `docker-compose` configuration for all developers on the same app, it avoids any port collisions and enables all developers to have a unified configuration.
+
+Using the above `the_second` sample project, we can create a file `~/the_second/.env` with this contents:
+
+```shell
+NGINX_HTTP_PORT=8081 
+NGINX_HTTPS_PORT=8444 
+CHT_COMPOSE_PROJECT_NAME=second 
+COUCHDB_SECRET=foo 
+DOCKER_CONFIG_PATH=./
+COUCHDB_DATA=./couchd 
+CHT_COMPOSE_PATH=./
+COUCHDB_USER=medic 
+COUCHDB_PASSWORD=password
+```
+
+Now it's easy to boot this environment:
+
+```shell
+cd ~/the_second
+docker-compose up
+```
+
+## Switching & concurrent projects
+
+The easiest way to switch between projects is to stop the first set of containers and start the second set. Cancel the first project running in the foreground with `ctrl + c` and `stop` all the project's services:
+
+```shell
+docker stop second_api_1 second_cht-upgrade-service_1 second_couchdb_1 second_haproxy_1 second_healthcheck_1 second_nginx_1 second_sentinel_1
+```
+
+Alternately, you can stop ALL containers (even non-CHT ones!) with `docker kill $(docker ps -q)`. Then start the other CHT project using either the `.env` file or use the explicit command with ports and other environment variables as shown above.
+
+To run projects concurrently, instead of cancelling and `stop`ing the first one, open a second terminal and start the second project.  Be careful you don't have port conflicts!
```

Comment:
```suggestion
To run projects concurrently open a second terminal and start the second project so you don't have to cancel and `stop` the first project.  Remember to avoid port conflicts!
```